src/psyclone/psyir/transformations/transformation_error.py

```diff
@@ -36,14 +36,12 @@
 '''This module provides the TransformationError class.
 '''
 
+from psyclone.errors import PsycloneError
 
-class TransformationError(Exception):
+class TransformationError(PsycloneError):
     ''' Provides a PSyclone-specific error class for errors found during
         code transformation operations. '''
 
     def __init__(self, value):
-        Exception.__init__(self, value)
-        self.value = "Transformation Error: "+value
-
-    def __str__(self):
-        return repr(self.value)
```

Comment:
Please could you add:

    # For AutoAPI documentation generation
    __all__ = ["TransformationError"]

to the bottom of this file.

Reply:
Thanks for doing this. Unfortunately I've discovered that we now get a truckload of warnings when building the reference guide I've asked a question on the sphinx user group (https://groups.google.com/g/sphinx-users/c/vuW6OOb96Yo) so we'll see what they say.

Reply:
Please could you just comment-out this `__all__` line for now with a comment to say that it causes 'more than one target for cross-reference' warnings when building the reference guide and add a TODO referring to #1280.